src/Frontend/Core.hs

```diff
@@ -346,8 +346,5 @@ redirectFormHandler getPage processor = getH :<|> postH
     renderer page v fa = FormPage page . toHtml . fmap (formPage v fa) <$> makeFrame page
 
 
---redirect :: (MonadError ActionExcept m) => ST -> m a
---redirect = Servant.Missing.redirect
-
```

Comment:
Hum here this was really an implicit FIXME of moving this to Servant.Missing and then one can use this redirect again.